File: 2-second-year-2022-2023/Programming-languages-and-laboratory/Python/Exercises/40_compito_python_20220620/Veicolo.py

class Veicolo:

    def __init__(self, codice, marca,  modello, targa,   costo):
        self._codice = codice
        self._marca =marca
        self._modello = modello
        self._costo =costo
        self._targa =targa

    def getCosto(self):
        return self._costo

    def getTarga(self):
        return self._targa

    def __str__(self):
        return str(self._targa) + "\t" +str(self._codice) + "\t" + self._modello + "\t" + self._marca + "\t" + str(
            self._costo) + "\t"

File: II - Linguaggi di programmazione e laboratorio/Esame/Java/40_compito_java_2020_06_22/Veicolo.java
import java.util.*;

public abstract class Veicolo {

    protected int codice;
    protected String targa, modello, marca;
    protected double costo;


    public Veicolo(int codice, String targa, String modello, String marca, double costo) {
        this.codice = codice;
        this.targa = targa;
        this.modello = modello;
        this.marca = marca;
        this.costo = costo;
    }

    public String getTarga() {
        return targa;
    }

    public int getCod() {
        return codice;
    }

    public String toString() {
        return targa + "\t" + codice + "\t" + modello + "\t" + marca + "\t" + costo + "\t";
    }
}